geopandas/geodataframe.py

```diff
@@ -441,9 +441,14 @@ def plot(self, *args, **kwargs):
     plot.__doc__ = plot_dataframe.__doc__
 
 
-    def dissolve(self, by=None, aggfunc='first'):
+    def dissolve(self, by=None, aggfunc='first', as_index=True):
         """
-        Dissolve geometries within `groupby` into single observation. 
+        Dissolve geometries within `groupby` into single observation.
+        This is accomplished by applying the `unary_union` method from the
+        `shapely` library to all geometries within a groupself.
```

Comment:
Maybe referring to shapely is not needed, as a GeoSeries has a unary_union as well (which is of course the shapely method)